test/test_helper.rb

```diff
@@ -2,10 +2,12 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'mocha/mini_test'
+require 'capybara/rails'
```

Comment:
This is already required in https://github.com/ruby-bench/ruby-bench-web/blob/master/test/acceptance/support/capybara.rb